### ISSUE TITLE
When clicking on a page to move the focus  on a line NVDA doesn't follow it

### ISSUE THREAD

**Issue by alexarnaud** [nvaccess/nvda#10935]
### Is your feature request related to a problem? Please describe.
Yes, without this feature added I can't recommend NVDA to low-vision users.

### Describe the solution you'd like
When a user click on a line on a web page it moves the caret on the line where he has clicked on. NVDA like the Orca sreen reader on Linux should be able to positioned the reading on line where the caret is.

### Describe alternatives you've considered
From a user perspective an alternative is to use the Narrator for this feature.

Best regards.

**Comment by Adriani90** [comment]
Closing as duplicate of #4896.
